src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    static final String COMMA_SEPARATOR = ",";
+    static final int OPERATION_TYPE = 0;
+    static final int AMOUNT = 1;
+    static final String SUPPLY = "supply";
+    static final String BUY = "buy";
+    static final String RESULT = "result";
+    static final int ZERO = 0;
```

Comment:
```suggestion
    static final String COMMA_SEPARATOR = ",";
    static final int OPERATION_TYPE_INDEX = 0;
    static final int AMOUNT_INDEX = 1;
```
if the name of constant is the same as string value - it doesn't make your code easier to read
zero is just a zero here - it has no other meaning, you should not add a constant for it

better use OPERATION_TYPE_**INDEX**, AMOUNT_**INDEX** because those are in fact indexes